pkg/dogstatsd/parser_test.go

```diff
@@ -631,3 +632,20 @@ func TestEventMetadataMultiple(t *testing.T) {
 	assert.Equal(t, "source test", e.SourceTypeName)
 	assert.Equal(t, "", e.EventType)
 }
+
+func TestCustomNamespace(t *testing.T) {
```

Comment:
Since we check the suffix of the namespace we should add a check with a namespace ending with a dot.